src/databricks/labs/ucx/install.py

```diff
@@ -213,6 +213,7 @@ def _prompt_for_new_installation(self) -> WorkspaceConfig:
         configure_groups = ConfigureGroups(self.prompts)
         configure_groups.run()
         include_databases = self._select_databases()
+        upload_dependencies = self.prompts.confirm("Does given workspace block Internet access?")
```

Comment:
Maybe at the workspace id/link. Especially useful for muilti workspace installation

Reply:
We have it printed earlier in the installation process, not sure if we have to

Reply:
have added workspace id in the prompt